org.openmw.OpenMW.json

```diff
@@ -0,0 +1,272 @@
+          "-DBUILD_OSG_APPLICATIONS=0",
+          "-DCMAKE_BUILD_TYPE=Release"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/OpenMW/osg/archive/b6ec7bb7a4cd06ae95bda087d48c0fb7d5ca0abf.zip",
+            "sha256": "12c4e2e4f96791751359379365d525717456f3ae435cf1c714b19ebe7775cf06"
+          }
+        ]
+      },
+      {
+        "name": "bullet",
+        /* Doesn't install the necessary files when using ninja */
+        "buildsystem": "cmake",
+        "config-opts": [
+          "-DBUILD_BULLET2_DEMOS=0",
+          "-DBUILD_BULLET3=0",
+          "-DBUILD_CPU_DEMOS=0",
+          "-DBUILD_EXTRAS=0",
+          "-DBUILD_OPENGL3_DEMOS=0",
+          "-DBUILD_UNIT_TESTS=0",
+          "-DCMAKE_BUILD_TYPE=Release",
+          "-DUSE_GLUT=0",
+          "-DUSE_GRAPHICAL_BENCHMARK=0"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/bulletphysics/bullet3/archive/2.86.1.tar.gz",
+            "sha256": "c058b2e4321ba6adaa656976c1a138c07b18fc03b29f5b82880d5d8228fbf059"
+          }
+        ]
+      },
+      {
+        "name": "mygui",
+        "buildsystem": "cmake-ninja",
+        "config-opts": [
+          "-DCMAKE_BUILD_TYPE=Release",
+          "-DMYGUI_RENDERSYSTEM=1",
+          "-DMYGUI_BUILD_DEMOS=0",
+          "-DMYGUI_BUILD_TOOLS=0",
+          "-DMYGUI_BUILD_PLUGINS=0"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/MyGUI/mygui/archive/MyGUI3.2.2.tar.gz",
+            "sha256": "0a28d7ec8a47993cb68deb48b36331e28f12dd92580b709eaef21d599b67a78f"
+          }
+        ]
+      },
+      {
+        "name": "libunshield",
+        "buildsystem": "cmake-ninja",
+        "config-opts": [
+          "-DCMAKE_BUILD_TYPE=Release"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/twogood/unshield/archive/1.4.2.tar.gz",
+            "sha256": "5dd4ea0c7e97ad8e3677ff3a254b116df08a5d041c2df8859aad5c4f88d1f774"
+          }
+        ]
+      },
+      {
+        "name": "boost",
+        "config-opts": [
+          "--with-libraries=system,filesystem,program_options"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2",
+            "sha256": "7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332"
+          },
+          {
+            "type": "script",
+            "commands": [
+              "./bootstrap.sh \"$@\"",
+              "cat <<EOF >Makefile",
+              "all:",
+              "\t./b2 -j `nproc` ./b2 stage threading=multi link=shared",
+              "",
+              "install:",
+              "\t./b2 install",
+              "EOF"
+            ],
+            "dest-filename": "configure"
```

Comment:
Just use the `simple` buildsystem. if you need an example, look at `net.minetest.Minetest`. (Also some apps like `org.gnode.NixView` already use boost so you can copy it from there.)